### DISCUSSION TITLE
ipynb files not rendering in MAC OS Safari but do render in Chrome

### DISCUSSION THREAD

**Discussion by essans** [community/community#65114] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello - Is there a reason why ipynb files are not rendering in my Mac OS desktop Safari but do render in Chrome ?  I've tried both private and public repos.  Thanks

**Comment by ghost** [comment]
@otter-computer @realkimpossible @essans <h1> I do not know</h1>

**Reply by ghost** [reply]
<h1>Yes I do not know</h1>

**Comment by lab596** [comment]
1. Clear Safari Cache:
Safari might be using cached data that is preventing the proper rendering of the .ipynb files. Clearing the cache can help resolve this issue.

2. Update Safari:
Ensure that you are using the latest version of Safari. Updates often include bug fixes and improvements that could resolve rendering issues.

3. Check Browser Compatibility:
Jupyter Notebook rendering might be more optimized for certain browsers. While Chrome is known for its compatibility with various web technologies, Safari might have certain limitations. Check if the official documentation or user forums provide any guidance on recommended browsers for rendering Jupyter Notebook files.

4. Browser Extensions and Settings:
Certain browser extensions or settings might interfere with the rendering of complex content like Jupyter Notebooks. Try disabling any extensions that might be affecting the rendering process.

5. Third-Party Renderers:
Consider using third-party rendering tools or extensions that are designed to improve Jupyter Notebook rendering on specific browsers. However, make sure these tools are reputable and well-maintained.

Documentation:

For further information and troubleshooting, you can refer to the official documentation and community resources related to Jupyter Notebook rendering and browser compatibility:

Jupyter Notebook Documentation: https://jupyter.org/documentation
GitHub Repository for Jupyter Notebook: https://github.com/jupyter/notebook

**Comment by ghost** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Hello - Is there a reason why ipynb files are not rendering in my Mac OS desktop Safari but do render in Chrome ? I've tried both private and public repos. Thanks

# **I do not [know.**]
[Omnifood]("https://omnifood.dev")

**Reply by lab596** [reply]
Lol what

**Comment by ghost** [comment]
![copy of Randomized Pizza](https://github.com/community/community/assets/125932707/884da98c-59cf-4460-a925-f6fc20ab966e)

**Comment by realliyifei** [comment]
Did you solve in the end? Same issue here

**Comment by keyuchen21** [comment]
same issues in 2024 lol
